test/util/benchmark_results.hpp

```diff
@@ -29,25 +29,20 @@ namespace test_util {
 
 class BenchmarkResults {
 public:
-    BenchmarkResults(int max_lead_text_width, const char* results_file_stem = "results");
+    BenchmarkResults(int max_lead_text_width, std::string suite_name, const char* results_file_stem = "results");
     ~BenchmarkResults();
 
-    enum ChangeType {
-        change_Percent,
-        change_DropFactor,
-        change_RiseFactor,
-    };
-
     /// Use submit_single() when you know there is only going to be a single datapoint.
-    void submit_single(const char* ident, const char* lead_text, double seconds, ChangeType = change_Percent);
+    void submit_single(const char* ident, const char* lead_text, std::string measurement_type, double seconds);
 
     /// Use submit() when there are multiple data points, and call finish() when you are done.
     void submit(const char* ident, double seconds);
-    void finish(const std::string& ident, const std::string& lead_text, ChangeType = change_Percent);
+    void finish(const std::string& ident, const std::string& lead_text, std::string measurement_type);
```

Comment:
the "measurement_type" here is what this metric will be called in the json file, so something like "latency" or "runtime" rather than the thing you're testing - which is captured in ident/lead_text.